fang/Cargo.toml

```diff
@@ -15,22 +15,29 @@ rust-version = "1.62"
 doctest = false
 
 [features]
-default = ["blocking", "asynk", "derive-error", "postgres", "mysql" , "sqlite", "migrations_postgres", "migrations_sqlite" , "migrations_mysql"]
-blocking = ["dep:diesel", "dep:diesel-derive-enum", "dep:dotenvy", "diesel?/chrono" , "diesel?/serde_json" , "diesel?/uuid"]
-asynk = ["dep:bb8-postgres", "dep:postgres-types", "dep:tokio", "dep:async-trait", "dep:async-recursion"]
+default = ["blocking", "asynk-sqlx", "derive-error", "blocking-postgres", "blocking-mysql" , "blocking-sqlite", "migrations-postgres", "migrations-sqlite" , "migrations-mysql"]
+asynk-postgres = ["asynk-sqlx" , "sqlx?/postgres"]
+asynk-sqlite = ["asynk-sqlx" , "sqlx?/sqlite"]
+asynk-mysql = ["asynk-sqlx"  , "sqlx?/mysql"]
+asynk-sqlx = ["asynk" , "dep:sqlx"]
+asynk = ["dep:tokio", "dep:async-trait", "dep:async-recursion" ]
 derive-error = ["dep:fang-derive-error"]
-postgres = ["diesel?/postgres" , "diesel?/serde_json", "diesel?/chrono" , "diesel?/uuid" , "diesel?/r2d2"]
-sqlite = ["diesel?/sqlite" , "diesel?/serde_json", "diesel?/chrono" , "diesel?/uuid" , "diesel?/r2d2"]
-mysql = ["diesel?/mysql" , "diesel?/serde_json", "diesel?/chrono" , "diesel?/uuid" , "diesel?/r2d2"]
-migrations_postgres = ["migrations"]
-migrations_sqlite = ["migrations"]
-migrations_mysql = ["migrations"]
+blocking = ["dep:diesel", "dep:diesel-derive-enum", "dep:dotenvy", "diesel?/chrono" , "diesel?/serde_json" , "diesel?/uuid", "diesel?/r2d2"]
+blocking-postgres = [ "blocking", "diesel?/postgres"]
+blocking-sqlite = ["blocking", "diesel?/sqlite" ]
+blocking-mysql = [ "blocking", "diesel?/mysql"]
+migrations-postgres = ["migrations"]
+migrations-sqlite = ["migrations"]
+migrations-mysql = ["migrations"]
```

Comment:
I think `migrations-*` features are not needed. since they only include `migrations`

Reply:
yes sir